src/components/tasks/CreateTask.jsx

```diff
@@ -0,0 +1,68 @@
+import { Fragment, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import Button from "src/components/button/Button.jsx";
+import { sanitizeInput } from "src/utils/helpers/utilities";
+import {
+  ALT_HIDE_BUTTON_TEXT,
+  CLASS_TASK_INPUT,
+  TEXT_ADD,
+  TEXT_ENTER,
+  TYPE_TEXT,
+} from "src/utils/constants/constants";
+import { ICON_BIN } from "src/utils/constants/imageSources";
+import "src/components/tasks/CreateTask.component.scss";
+
+const CreateTask = ({ onAddTask, onHideButtonClick }) => {
```

Comment:
Can we maintain consistency while writing callback props ? [reference](https://github.com/azo-wtag/todo-app-react/pull/2#discussion_r1204980641)
```suggestion
const CreateTask = ({ onAddTask, onHideTask }) => {
```